Update Program.cs

saberthesword opened the pull request:
changed the firewood names to the latest plugin

saberthesword committed:
Update Program.cs

changed the firewood names to the latest plugin

Noggog commented:
I'm guessing #3 and #4 are sort of intertwined and sort of the same PR?

can you give some more insight on why this change is being done?   Just a new esp the original author came out with im guessing?

saberthesword commented:
yeah, a few changes, actually. also first time using github for anything that wasn't reading other peoples' code hahaha.

1. the last update changed it from an esp to an esl flagged esp file, which changed all the formIDs
2. the editorIDs of the firewood were also changed
3. one of the firewood has a proper replacement so i updated it to replace that particular mesh

Noggog commented:
Awesome!  Let me know if you want any 101s interacting with all this.    Content upgrade looks fine enough to me from a glance
